cmd/validate.go

```diff
@@ -66,6 +79,60 @@ this command.
 	},
 }
 
+func validate(ctx context.Context, entity interface{}, kongClient *kong.Client, entityType string) string {
+	_, err := performValidate(ctx, kongClient, entity, entityType)
+	output := ""
+	if err != nil {
+		output = color.New(color.FgRed, color.Bold).Sprintf("%s: %v", entityType, err)
+	}
+	return output
+}
+
+func validateEntities(ctx context.Context, obj interface{}, kongClient *kong.Client, entityType string) {
+	// call GetAll on entity
+	method := reflect.ValueOf(obj).MethodByName("GetAll")
+	entities := method.Call([]reflect.Value{})[0].Interface()
```

Comment:
Since there are no tests, we will need to figure out a way to catch an error here.
Can you use `init()` function to invoke code like this to ensure that it doesn't error/panic?

Reply:
Not sure if I got you here, can you elaborate a bit please?

We can also do a bit more of checks like this:

```
method := reflect.ValueOf(obj).MethodByName("GetAll")
	if !method.IsValid() {
		// bla
	}
	entities := method.Call([]reflect.Value{})[0].Interface()
	values := reflect.ValueOf(entities)
	if values.Kind() != reflect.Slice {
		// bla
	}
```